UPGRADE-2.8.md

```diff
@@ -546,6 +546,9 @@ Security
 
  * The `VoterInterface::supportsClass` and `supportsAttribute` methods were
    deprecated and will be removed from the interface in 3.0.
+   
+ * The the `key` setting of `anonymous`, `remember_me` and `http_digest`
+   is deprecated, and will be removed in 3.0.  Use `secret` instead.
```

Comment:
Okaydoke, and thanks for the link.  Added in 84a6d70bf9d4b5949981623c42bcb580e6846c67.